packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -494,12 +495,24 @@ export default class Meetings extends WebexPlugin {
      * @memberof Meetings
      */
     async createMeeting(destination, type = null) {
+      let targetDest;
```

Comment:
Super nitpick but you could default to `let targetDest = destination;` and remove lines 505:508

Reply:
implemented with a tertiary.